test/functional/ui/popupmenu_spec.lua

```diff
@@ -1716,28 +1717,42 @@ describe('builtin popupmenu', function()
             autocmd! Group
             autocmd CompleteChanged * call TsHl()
           endfunc
+          funct Append_multipe()
```

Comment:
what is "pe" in "multipe" ?

Reply:
Let a completeitem have multiple lines of info text to test the second selection of this item. The info text is not truncated. TsHl also adds a multi-line info text, but nvim__complete_set does not store this info. It is only added to the preview buffer.

Reply:
oh, just a typo then? `Append_multiple`